src/main/java/app/plugins/greeting/GreetingPlugin.java

```diff
@@ -0,0 +1,13 @@
+package app.plugins.greeting;
+
+import app.IPlugin;
+
+/**
+ * Created by Ihar_Rubanovich on 2/22/2017.
+ */
+public class GreetingPlugin extends AUtils implements IPlugin {
+    public void doAction() {
+        AUtils ut= new AUtils();
+        ut.show();
```

Comment:
you can just call this.show() since you extend AUtils, you don't need new object here